client/src/components/profile.ts

```diff
@@ -602,7 +599,8 @@ const input: ProfilePromptInput = {
   [ProfilePromptType.ClientId]: {
     title: "Client ID",
     placeholder: "Enter a client ID",
-    description: "Enter the registered client ID. An example is myapp.client.",
+    description:
+      "Enter the registered client ID. An example is myapp.client. For SAS Viya 2022.11 and later, you can leave Client ID empty and simply press Enter",
```

Comment:
If we were to make a change, having something like this would be preferable...
```suggestion
      "Enter the registered client ID. An example is myapp.client. For SAS Viya 2022.11 and later, leave client ID empty.",
```
(since `Press 'Enter'...` gets appended to the selection anyway).

@skyeaussie Do we think this additional context is worth having in the profile prompt?

Reply:
@scottdover here https://github.com/sassoftware/vscode-sas-extension/blob/main/README.md#add-new-sas-profile means that you can leave the prompt empty, but if you do, editing will be aborted (_delete profileClone.clientId;_). 

https://github.com/sassoftware/vscode-sas-extension/blob/a4f53667babf06666fabc708646223e7a241a1a8/client/src/components/profile.ts#L425-L432

Reply:
Interesting. My understanding is that `delete profileClone.clientId` only suppresses clientId from being written to user settings

Reply:
@scottdover I don't know how it works. I've tried in several VS and each time it breaks the edit/add process if I try to proceed without specifying clientId.

Reply:
@tsej, what did you mean by interrupted? If you leave clientid empty and press Enter, the profile creation process will just complete. Default client id doesn't need to be written to your profile. Do you have any error connecting with the created profile?